bower.json

```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/PolymerElements/paper-input",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.2.0",
```

Comment:
I don't have to change it. I think I was being overly pedantic, maybe. Should I revert it?